src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -1,38 +1,51 @@
 package nextstep.ladder.domain;
 
-import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
 public class Ladder {
-    private final int height;
-    private final List<Line> lines = new ArrayList<>();
+    private final JoinMembers joinMembers;
+    private final List<Line> lines;
 
-    public Ladder(int height, int countOfPerson) {
-        this(height, IntStream.range(0, height)
-                .mapToObj((i) -> new Line(countOfPerson))
+    public Ladder(JoinMembers joinMembers, int height) {
```

Comment:
```suggestion
    public Ladder(String names, int height) {
```

이 생성자도 부생성자로 추가하는 것은 어떨까?

Reply:
그게 좋겠네요! 추가했습니다!